pkg/clusteragent/admission/metrics/metrics.go

```diff
@@ -60,10 +60,10 @@ var (
 		telemetry.Options{NoDoubleUnderscoreSep: true},
 	)
 	LibInjectionAttempts = telemetry.NewCounterWithOpts("admission_webhooks", "library_injection_attempts",
-		[]string{"language", "injected", "auto_detected"}, "Number of pod library injection attempts by language.",
+		[]string{"language", "injected", "auto_detected", "injection_type"}, "Number of pod library injection attempts by language.",
```

Comment:
nice to have: the comment could reflect the change too. 

```suggestion
		[]string{"language", "injected", "auto_detected", "injection_type"}, "Number of pod library injection attempts by language and injection type",
```